corehq/apps/hqcase/api/get_list.py

```diff
@@ -0,0 +1,96 @@
+import datetime
+from itertools import chain
+
+from dateutil.parser import parse
+
+from dimagi.utils.parsing import FALSE_STRINGS
+
+from corehq.apps.es import case_search
+from corehq.apps.es import cases as case_es
+
+from .core import UserError, serialize_es_case
+
+DEFAULT_PAGE_SIZE = 20
+MAX_PAGE_SIZE = 5000
+CUSTOM_PROPERTY_PREFIX = 'property.'
+
+
+def _to_boolean(val):
+    return val.lower() not in [''] + list(FALSE_STRINGS)
```

Comment:
nit: slightly more efficient
```suggestion
    return val != '' and val.lower() not in FALSE_STRINGS
```
I find it easier to understand with the logic reversed:
```suggestion
    return not (val == '' or val.lower() in FALSE_STRINGS)
```

Reply:
Works for me.